rust/src/web/graphql/stakes/mod.rs

```diff
@@ -2,29 +2,39 @@ use crate::{
     ledger::{staking::StakingAccount, store::LedgerStore},
     store::IndexerStore,
 };
-use async_graphql::{Context, InputObject, Object, Result, SimpleObject};
+use async_graphql::{Context, Enum, InputObject, Object, Result, SimpleObject};
 use rust_decimal::{prelude::ToPrimitive, Decimal};
 use std::sync::Arc;
 
 #[derive(InputObject)]
-pub struct StakesQueryInput {
+pub struct StakeQueryInput {
     epoch: u32,
 }
 
+#[derive(Enum, Copy, Clone, Eq, PartialEq)]
+pub enum StakeSortByInput {
+    BalanceDesc,
+}
+
```

Comment:
Does this need [SCREAMING_SNAKE_CASE](https://github.com/async-graphql/async-graphql/blob/master/src/docs/enum.md)?

<img width="763" alt="image" src="https://github.com/Granola-Team/mina-indexer/assets/9849069/ec90de41-7c21-4ff5-b074-8be650723efd">

```suggestion
#[derive(Enum, Copy, Clone, Eq, PartialEq)]
#[graphql(rename_items = "SCREAMING_SNAKE_CASE")]
pub enum StakeSortByInput {
    BalanceDesc,
}

```